doc/plutus/howtos/analysing-scripts.rst

```diff
@@ -58,7 +58,7 @@ results in the following output:
     Writing script: ./tmp/auction_2-5.flat (Size: 9.1kB, Cost: ExCPU 1126876612, ExMemory 3408894)
 
 .. note::
-    The program writes out fully applied validators by default. Fully applied validators are larger than unapplied validators because they contain not just the validator code itself but also all arguments, including the :hsobj:`Plutus.V1.Ledger.Contexts.ScriptContext`. The script context can be quite large as it is a representation of the entire transaction body.
```

Comment:
`Plutus.V1.Ledger.Contexts.ScriptContext` is nowhere to be found in `plutus` nor `plutus-apps`.
Also there are many such instances in the `./doc` folder.
There are references to `Plutus.V1.*` and `Plutus.V2.*` modules which simply do not exist in `plutus` nor `plutus-apps`.

Reply:
They still exist in the version of plutus that we use. I guess they will be fixed when we migrate to a newer version?

Reply:
It think it'll change to e.g. `PlutusLedgerApi.V1.Contexts.ScriptContext`

Reply:
> They still exist in the version of plutus that we use. I guess they will be fixed when we migrate to a newer version?

If mean the version of the plutus _flake_ that is used in plutus-apps?
I checked out that commit `7c3b4faebc8da65b5bb8c771882fe4155d691990` but still searching for `module Plutus.V1.*` yields no results.

However `PlutusLedgerApi.V1.*` can be found. 
I'll replace those and see how far I can get

Reply:
Something weird is going on. We're still using `Plutus.V1.*` and `Plutus.V2.*` imports all over the place!

Reply:
I've replaced all occurrences of `Plutus.V1.Ledger.Api` with `PlutusLedgerApi.V1`

Reply:
It appears that the haddock used by the `read-the-docs` site comes from a version of plutus that is different from the one used in cabal.project.

`cabal.project`
- `1.0.0.1 | Tue Sep 27 13:14:40 2022 | d5f6f4a5505094490f61c8d164515adf7c8f46cc`
- See https://github.com/input-output-hk/cardano-haskell-packages/blob/main/_sources/plutus-core/1.0.0.1/meta.toml

`read-the-docs` (coming from flake.nix -> see flake.lock)
- `Fri Feb 24 18:35:46 2023 | 7c3b4faebc8da65b5bb8c771882fe4155d691990`

I've updated the haskell references inside ./doc, though there are 3 of them for which I need help (see the comments in this diff)